crates/ibc/src/erased.rs

```diff
@@ -0,0 +1,6 @@
+#[cfg(feature = "serde")]
+pub use erased_serde::Serialize as ErasedSerialize;
+#[cfg(not(feature = "serde"))]
```

Comment:
Can you explain what we're doing here in a comment?

Reply:
https://github.com/cosmos/ibc-rs/pull/331/commits/1d5e2fbc9e6595439ae05b719d39dcf1d2313414